pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -464,27 +465,50 @@ func applyUpstreamTLSSettings(cluster *v2.Cluster, tls *networking.TLSSettings)
 				cluster.Name)
 			return
 		}
+
 		cluster.TlsContext = &auth.UpstreamTlsContext{
-			CommonTlsContext: &auth.CommonTlsContext{
-				TlsCertificates: []*auth.TlsCertificate{
-					{
-						CertificateChain: &core.DataSource{
-							Specifier: &core.DataSource_Filename{
-								Filename: tls.ClientCertificate,
-							},
+			CommonTlsContext: &auth.CommonTlsContext{},
+			Sni:              tls.Sni,
+		}
+
+		if sdsUdsPath == "" {
+			log.Debuga("SDS isn't enabled")
```

Comment:
Can you remove this? This is going to be super noisy even in debug. For each service and cluster variant.

Reply:
removed